windows.md

```diff
@@ -312,6 +312,26 @@ Before you start, please make sure that you're running Windows 10 or Windows 11.
     It should print out a welcome message like this:<br><br>
     <img src="macos-6-docker-hello-world.png"><br><br>
 
+    If after running `docker run hello-world` command, you receive an error like this:
+
+    ```bash
+    Unable to write to the database. Exit code: 1
+      at Docker.Backend.Database.WriteKeys(ICollection\'1 keyValues) at Docker.Backend.Database.ModifyDatabase(ICollection`1 keyValuesToWrite, ICollection`1 keysToRemove)
+      at Docker.Backend.Database.Write(Settings settings)
+      at Docker.Backend.ContainerEngine.Linux.DoStart(Settings settings)
+      at Docker.Backend.ContainerEngine.Linux.Start(Settings settings)
+      at Docker.Core.Pipe.NamedPipeServer.<>c**DisplayClass8_0.<Register>b**0(Object[] parameters)
+      at Docker.Core.Pipe.NamedPipeServer.RunAction(String action, Object[] parameters)
+    ```
+
+    1. Press <kbd>Ctrl</kbd>-<kbd>Alt</kbd>-<kbd>Delete</kbd> and click on Task Manager
+       <img src="windows-7-task-manager.webp"><br><br>
```

Comment:
this image is placed in an unusual place and doesn't really add anything to the instructions

![Screen Shot 2022-04-30 at 12 57 50](https://user-images.githubusercontent.com/1935696/166102692-d303119e-ce6b-449d-a311-35547d422115.png)

Reply:
removed in 19ef4d8